### ISSUE TITLE
is there a way to use DSL AC68U as AImesh node or repater with wired backhaul connection?

### ISSUE THREAD

**Issue by junkkari3** [gnuton/asuswrt-merlin.ng#629]
I hope this is the right way to open the ticket. I request a change to firmware which allows DSL AC68U to work as AImesh node wired backhaul connection? Thank you for the great firmware development you do!

### Discussed in https://github.com/gnuton/asuswrt-merlin.ng/discussions/626

<div type='discussions-op-text'>

<sup>Originally posted by **junkkari3** July  9, 2024</sup>
I have two DSL AC68U which I would like to add as wired AI mesh nodes or repeaters. Is it possible? When they are in router mode, I can configure so that instead of the DSL port, it uses one of the four "ethernet out" (lan or what ever) ports as "ethernet in" port (wan or what ever). However, I do not want to use them in router mode, I want to add them to my existing wifi set up (router and mesh node) as mesh nodes, or as repeaters - with wired backhaul connection. Is this somehow possible with the latest gnuton merlin firmware, or with the (old) experimental repeater firmware? I.e. configuring the device in AI node / repeater mode to use wired backhaul (because of the DSL port this is not possible in the ASUS standard software).  Thank you anyway, for a long while I used these two devices with your built through the wireless connection. The build was great (but the devices started acting up so I bought new ones, and now want to add these as wired nodes)</div>

**Comment by gnuton** [comment]
Investigation:

Dual wan enabled:
```gnuton@DSL-AC68U-CC88:/tmp/home/root# nvram get wans_dualwan
lan dsl
gnuton@DSL-AC68U-CC88:/tmp/home/root# nvram get wan_ifnames
vlan4 vlan3880
gnuton@DSL-AC68U-CC88:/tmp/home/root# nvram get wandevs
vlan3880 vlan4
gnuton@DSL-AC68U-CC88:/tmp/home/root# nvram get lan_ifnames
vlan1 eth1 eth2 wl0.1```

LAN1 as WAN
gnuton@DSL-AC68U-CC88:/tmp/home/root# nvram get wans_dualwan
lan none
gnuton@DSL-AC68U-CC88:/tmp/home/root# nvram get wan_ifnames
vlan4
gnuton@DSL-AC68U-CC88:/tmp/home/root# nvram get wandevs
et0
gnuton@DSL-AC68U-CC88:/tmp/home/root# nvram get lan_ifnames
vlan1 eth1 eth2 wl0.1
```

**Comment by FastBoot3DS** [comment]
> Investigation:
> 
> Dual wan enabled:
> 
> ```
> lan dsl
> gnuton@DSL-AC68U-CC88:/tmp/home/root# nvram get wan_ifnames
> vlan4 vlan3880
> gnuton@DSL-AC68U-CC88:/tmp/home/root# nvram get wandevs
> vlan3880 vlan4
> gnuton@DSL-AC68U-CC88:/tmp/home/root# nvram get lan_ifnames
> vlan1 eth1 eth2 wl0.1
> ```
> 
> LAN1 as WAN
> 
> ```
> gnuton@DSL-AC68U-CC88:/tmp/home/root# nvram get wans_dualwan
> lan none
> gnuton@DSL-AC68U-CC88:/tmp/home/root# nvram get wan_ifnames
> vlan4
> gnuton@DSL-AC68U-CC88:/tmp/home/root# nvram get wandevs
> et0
> gnuton@DSL-AC68U-CC88:/tmp/home/root# nvram get lan_ifnames
> vlan1 eth1 eth2 wl0.1
> ```

Hi there just wondering is this done with the repeater firmware or just the latest merlin release for the DSL-AC68U

I would like to setup this up just wondering whats the best way of doing this?

**Comment by gnuton** [comment]
@Kai-Denzel-Jane what are you trying to achieve?

**Comment by FastBoot3DS** [comment]
> [@Kai-Denzel-Jane](https://github.com/Kai-Denzel-Jane) what are you trying to achieve?

Use Ethernet Backhaul with the DSL-AC68U

**Comment by gnuton** [comment]
@Kai-Denzel-Jane that'\s not supported, but I guess it may be possible also with the lastest image (not repeater)

**Comment by FastBoot3DS** [comment]
> [@Kai-Denzel-Jane](https://github.com/Kai-Denzel-Jane) that'\s not supported, but I guess it may be possible also with the lastest image (not repeater)

Yeah I was thinking if you set one of the LAN Ports to WAN surely it might work.

**Comment by gnuton** [comment]
@Kai-Denzel-Jane I didn't test that, but yes it may work.
you can play with nvram to test this, but you can also set these in the router/rc/init.c 
NOTE: By changing this you may also need to change several other parts to make this changes correcly visible on the web ui too.

**Comment by Harrenkyym** [comment]
So it is or it is not supported? I didn't understand what you did on that message

> Investigation:
> 
> Dual wan enabled:
> 
> ```
> lan dsl
> gnuton@DSL-AC68U-CC88:/tmp/home/root# nvram get wan_ifnames
> vlan4 vlan3880
> gnuton@DSL-AC68U-CC88:/tmp/home/root# nvram get wandevs
> vlan3880 vlan4
> gnuton@DSL-AC68U-CC88:/tmp/home/root# nvram get lan_ifnames
> vlan1 eth1 eth2 wl0.1
> ```
> 
> LAN1 as WAN
> 
> ```
> gnuton@DSL-AC68U-CC88:/tmp/home/root# nvram get wans_dualwan
> lan none
> gnuton@DSL-AC68U-CC88:/tmp/home/root# nvram get wan_ifnames
> vlan4
> gnuton@DSL-AC68U-CC88:/tmp/home/root# nvram get wandevs
> et0
> gnuton@DSL-AC68U-CC88:/tmp/home/root# nvram get lan_ifnames
> vlan1 eth1 eth2 wl0.1
> ```

that makes ethernet backhaul available?

is it a real option? or today it is just possible to connect it wireless?

Grazie. :)

**Comment by gnuton** [comment]
@Harrenkyym i have never investigated this properly, but i feel like it should be possible if there is nothing hardcoded in the prebuilt binaries. You can explore this by changing the nvram and restarting few daemons/interfaces or the router itself.
